### DISCUSSION TITLE
TF1 Channels

### DISCUSSION THREAD

**Discussion by GiovanniSebastian** [iptv-org/community#129] [Show and tell]
Hello, I found the channels of the French network TF1 and it gives me a file that can be opened with Notepad, but I cannot open it with IPTV, because it is a file with a token.

These are:
TF1: https://rebrand.ly/t/f/1/o/f/f
TFX: https://rebrand.ly/t/f/x/o/f/f
TMC: https://rebrand.ly/t/m/c/o/f/f
TF1 Séries Films: https://rebrand.ly/t/s/f/o/f/f

It looks like the channels come from this page:
http://azrotv.com/iphone/Fr/ch2/t1.php

**Comment by what-a** [comment]
These were working, but 12 hours later all are dead. Must be a time limit.
TF1:
http://azrotv.com/m3u8/pplayer/fra.php/https://tf1-hls-live.tf1.fr/video/Bd2lD7gOWwzhslAGTKqrPQ/1651653320/out/v1/c2e382be3aa2486e8753747e7bb6157e/index.m3u8
TFX:
http://azrotv.com/m3u8/pplayer/fra.php/https://tfx-hls-live.tf1.fr/video/ZV7CalGFsgkxHayT0r3tEg/1651653581/out/v1/818e38d791854222b71a950791fb7002/index.m3u8
TMC:
http://azrotv.com/m3u8/pplayer/fra.php/https://tmc-hls-live.tf1.fr/video/qP1lpnJ_S41e0ZaEux_YQA/1651653634/out/v1/982927d0b5d24bd58ac42184b10c63d7/index.m3u8
TF1 Séries Films:
http://azrotv.com/m3u8/pplayer/fra.php/https://tsf-hls-live.tf1.fr/video/UOBA2mPRM-P8cJxK5Mfq1Q/1651653483/out/v1/a5def86fc7db4dbc9ac922f15a55d4c9/index.m3u8

**Reply by BellezaEmporium** [reply]
> These were working, but 12 hours later all are dead. Must be a time limit. TF1: http://azrotv.com/m3u8/pplayer/fra.php/https://tf1-hls-live.tf1.fr/video/Bd2lD7gOWwzhslAGTKqrPQ/1651653320/out/v1/c2e382be3aa2486e8753747e7bb6157e/index.m3u8 TFX: http://azrotv.com/m3u8/pplayer/fra.php/https://tfx-hls-live.tf1.fr/video/ZV7CalGFsgkxHayT0r3tEg/1651653581/out/v1/818e38d791854222b71a950791fb7002/index.m3u8 TMC: http://azrotv.com/m3u8/pplayer/fra.php/https://tmc-hls-live.tf1.fr/video/qP1lpnJ_S41e0ZaEux_YQA/1651653634/out/v1/982927d0b5d24bd58ac42184b10c63d7/index.m3u8 TF1 Séries Films: http://azrotv.com/m3u8/pplayer/fra.php/https://tsf-hls-live.tf1.fr/video/UOBA2mPRM-P8cJxK5Mfq1Q/1651653483/out/v1/a5def86fc7db4dbc9ac922f15a55d4c9/index.m3u8

TF1 link grabbing works with Streamlink (my solution works), if the IP is based in France only.
